client/py/synnax/auth.py

```diff
@@ -100,6 +101,17 @@ def authenticate(self) -> None:
         self.token = res.token
         self.user = res.user
 
+    def register(self, username: str, password: str) -> UserPayload:
```

Comment:
This feels like it probably belongs inside of the `user` module instead, so we access it with `client.users.register` or something along those lines.

Reply:
We don't have a user module – in both the server and the client the user module is the same as the auth module

Reply:
Should we maybe just rename the `auth` module to `user`?

Reply:
Well the `auth` service and the `user` service are different in the server, which they should be. Maybe instead we should split the `auth` API in the server?